library/entity/entity-events/build.gradle

```
@@ -0,0 +1,12 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	moduleName = "entity-events"
```

```suggestion
	moduleName = "entity_events"
```

QSL libraries and modules are snake_case.